.circleci/config.yml

```diff
@@ -16,6 +16,10 @@ jobs:
           keys:
           - api-cache-{{ checksum "~/workbench/api/build.gradle" }}
           - api-cache-
+      - run:
+          name: Validate swagger definitions
+          working_directory: ~/workbench/api
+          command: gradle validateSwagger --project-prop verboseTestLogging=yes
```

Comment:
Another option here would be to add a command to our ruby scripts. If you copy the structure of the "test" command, you could change this so the command is `./project.rb validate-swagger --project-prop verboseTestLogging=yes` and it would serve as an example of exactly what developers need to run if they want to validate when they make changes to the Swagger file.

Reply:
Done.